testing/mock/test_system.cpp

```diff
@@ -736,6 +738,30 @@ int test_system::glob(const char *pattern, int flags,
   return res;
 }
 
+char *test_system::realpath(const char *inp, char *dst)
```

Comment:
dst will need to be freed if it's NULL when mock test calls this. 👍

Reply:
It's up to the caller what dst is. Typically it will be an array allocated on the stack.